Dockerfile

```diff
@@ -0,0 +1,39 @@
+# docker image build --build-arg SOURCE_COMMIT=`git rev-parse --verify HEAD` .
+# docker build --build-arg SOURCE_COMMIT=`git rev-parse --verify HEAD` -t harbor.intgdc.com/adam.stulpa/hello-world-brick:latest .
+FROM harbor.intgdc.com/staging/gdc-java-8-jre:1.8.0.171-8.b10
+
+MAINTAINER tma <tma@gooddata.com>
+
+LABEL image_name="GDC LCM Bricks"
+LABEL maintainer="TMA <tma@gooddata.com>"
+LABEL git_repostiory_url="https://github.com/gooddata/gooddata-ruby/"
+LABEL parent_image="harbor.intgdc.com/base/centos:7.5.1804"
```

Comment:
I agree, but this label is required because it's used in build pipeline. See https://confluence.intgdc.com/display/plat/Kubernetes+Developers+Guideline or ask PGR for more details.